src/Exports/DonorsExport.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+namespace Give\Exports;
+
+use DateTime;
+use Give\Framework\Database\DB;
+use Give\Framework\QueryBuilder\JoinQueryBuilder;
+use Give_Batch_Export;
+
+/**
+ * @unreleased
+ */
+class DonorsExport extends Give_Batch_Export
+{
+    /** @inheritdoc */
+    public $export_type = 'donors';
+
+    /** @inheritdoc */
+    protected $posted_data;
+
+    /** @var DateTime */
+    protected $startDate;
+
+    /** @var DateTime */
+    protected $endDate;
+
+    /** @var String */
+    protected $searchBy;
+
+    /**
+     * @inheritdoc
+     */
+    public function set_properties( $posted_data ) {
+        $this->posted_data = $posted_data;
+
+        if( $this->posted_data['start_date'] ) {
+            $this->startDate = date('Y-m-d', strtotime($this->posted_data['start_date']));
+        }
+
+        if( $this->posted_data['end_date'] ) {
+            $this->endDate = date('Y-m-d', strtotime($this->posted_data['end_date']));
+        }
```

Comment:
I noticed the dates were not doing anything.  It appears the field names changed to `giveDonorExport-startDate` and `giveDonorExport-endDate` but were not updated here.  So these will need to be updated or revert the front-end ids w/e makes more sense.